packages/react-vapor/src/components/color/Color.tsx

```diff
@@ -11,23 +10,16 @@ export const VaporColors: Record<string, string> = webpackVaporColors;
 
 export class Color extends React.Component<ColorProps & React.HTMLProps<HTMLDivElement>> {
     static defaultprops: Partial<ColorProps> = {
-        color: 'pure-white',
+        color: '#FFFFFF', // var(--white)?

```

Comment:
I think all the places where you put the hex and the var should only have one.

If you need help using the variable ask in slack 😄